mitodl/mitxpro-theme
# style: update studio logo HD

### What are the relevant tickets?
None

### Description (What does it do?)
Noticed that the logo on the studio had a blurry effect. In this PR I replaced it with a good quality one.

### Screenshots (if appropriate):
**Before**
<img width="1792" alt="image" src="https://github.com/user-attachments/assets/58415cdb-c514-4f3b-a05d-2ff55ce539e0">


**After**
<img width="1789" alt="image" src="https://github.com/user-attachments/assets/33bd76af-eb89-4a75-b885-24da6236603c">

### How can this be tested?
-  Set up the theme, Open studio. You should be able to see a good quality logo there


**NOTE:**

I'll create a backport of this PR to our `quince` branch as well.
